### ISSUE TITLE
[EN][Quest][Fellwood]Ruumbo Demands Honey

### ISSUE THREAD

**Issue by Thefakemccoy** [Atlantiss/BugTracker#6527]
The honey hives drop down and into the ground.
Also when you are attacked by the bees the aura says jump into water to remove it didn't remove it for me.
http://www.wowhead.com/quest=27989/ruumbo-demands-honey

**Comment by karhima** [comment]
Yes, even /tar them doesn't work.

**Comment by purabasura666** [comment]
Still bugged as of Oct 27, 2017. Visited the area a few times over the past few days, and only found one honey hive. Today, shortly after a server crash, some had respawn, and could complete the quest. Actually a nice GM teleported me to the area. 

So it seems that the problem persists, and the quest is only doable after a server restart. 

It'd be nice to have it fixed, because many quests are unlocked only once this one is completed. 

Thanks!!!

**Comment by MercuricUma** [comment]
I've just completed it by using the quest item under the hives even without seeing them :) Gave it a try and worked out
